stylesheet/default.css

```diff
@@ -0,0 +1,101 @@
+/* reset */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+/* font size */
+html {
+  font-size: 100%;
+  scroll-behavior: smooth;
+}
+body {
+  font-size: 1.2rem;
+}
+
+/* colors */
+:root {
+  --correct-color: #2bc20e;
+  --incorrect-color: #cb0000;
+  --background-color: #0e131f;
+  --background-S-color: #f2f7f2;
+  --extra-color: #e0acd5;
+}
+
+/* gap // spacing */
+:root {
+  --gap-largest: 2rem 3.5rem;
+  --gap-large: 1.5rem 2.5rem;
+  --gap-medium: 1rem 1.5rem;
+  --gap-small: 0.5rem 1rem;
+  --gap-smallest: 0.25rem 0.5rem;
+}
+
+/* border radius */
+
+:root {
```

Comment:
why multiple `:root` you can create a single root and use variables inside it

Reply:
yes i will do that